packages/fxa-admin-panel/server/lib/csp/blocking.ts

```diff
@@ -35,6 +36,13 @@ export default function cspBlocking(config: { [key: string]: any }) {
     return target;
   }
 
+  let styleSrc = [SELF];
```

Comment:
nit: could narrow the type of `styleSrc` to `("'self'" | "'unsafe-inline'")[]` so it's not simply `string[]`.